src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClientCertificateManager.cs

```diff
@@ -29,6 +29,8 @@ public static class BridgeClientCertificateManager
         private const string ClientCertificateSubject = "WCF Client Certificate";
         private const string ClientCertificatePassword = "test"; // this needs to be kept in sync with the Bridge configuration 
 
+        public static string Thumbprint { get; private set; }
```

Comment:
+1 to naming. 

The cert should be consistent across one test run (i.e., one instance of the Bridge and configuration). If set by multiple tests, should mean that we continue to set it to the same value.